src/Orleans/Placement/PlacementTarget.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Orleans.Core;
+
+namespace Orleans.Runtime.Placement
+{
+    public struct PlacementTarget
+    {
+        public IGrainIdentity GrainIdentity { get; }
+
+        public int InterfaceId { get; }
+
+        public ushort InterfaceVersion { get; }
+
+        public bool IsClient => GrainIdentity.IsClient;
+
+        public PlacementTarget(IGrainIdentity grainIdentity, int interfaceId, ushort interfaceVersion)
+        {
+			GrainIdentity = grainIdentity;
```

Comment:
Weird. Must be different setting on my desktop. I fixed them, but Git seems to be ignoring the changes.